test/manifests.go

```diff
@@ -407,6 +455,17 @@ const UpdateManifest = `{
 	}
 }`
 
+// SecretsManifest is a test secrets manifest to update secrets
+const SecretsManifest = `{
```

Comment:
Maybe we shouldn't name all json documents manifest. We might only name things "SomethingManifest" if they keep the same format as the Manifest like the UpdateManifest does. Opinions? @m1ghtym0 @Nirusu

Reply:
Yep, agreed.

Reply:
Originally this data structure was directly marshaled into a `manifest.Manifest` object. 
I agree that at the current state it would probably be better renaming it to something simpler like `UserSecrets`